src/components/LoanCards/LendHomepageLoanCard.vue

```diff
@@ -212,173 +167,94 @@ export default {
 
 <style lang="scss" scoped>
 @import "settings";
-@import "components/loan-cards/hover-loan-card";
-
-.hover-loan-card-large {
-	@extend .base-portrait-hover-loan-card;
-
-	width: $large-hover-card-width;
-	height: $large-hover-card-height;
-	// transition: $hover-card-transition-transform, $hover-card-transition-opacity-out;
-	transition: $hover-card-transition-transform, $hover-card-transition-opacity-out;
-	z-index: 1;
-
-	.hover-loan-card-image {
-		@extend .base-portrait-hover-loan-card-image;
 
-		height: $large-hover-card-image-height;
+// These values have to be the same as the values in src/components/Homepage/LendByCategory/LoanCategory.vue
+$card-width: rem-calc(305);
+$card-margin: rem-calc(14);
+$card-half-space: rem-calc(14/2);
+$card-height: rem-calc(388);
+
```

Comment:
Should we define these in _settings.scss, then have the variable to use here and in LendHomepageLoanCards.vue? There's a card definition grouping on line 557 of _settings.scss